### PR TITLE
Update fdo to 21.08

### PR DESCRIPTION
**Pull request by A6GibKm**

### CODE REVIEW

File: cc.arduino.arduinoide.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "app-id": "cc.arduino.arduinoide",
     "command": "arduino",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "20.08",
+    "runtime-version": "21.08",
     "sdk": "org.freedesktop.Sdk",
     "sdk-extensions": [
         "org.freedesktop.Sdk.Extension.openjdk8"
```
Comment (MartinIIOT) [inline cc.arduino.arduinoide.json:8]:
```suggestion
        "org.freedesktop.Sdk.Extension.openjdk11"
```

**Review by MartinIIOT — CHANGES_REQUESTED** [review]
Try update openJDK to 11 and make changes in lines 63, 64 too.

**Comment by MartinIIOT** [comment]
OpenJDK 8 is available for 21.08 runtime:
https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk8/issues/13#issuecomment-939047080

**Comment by A6GibKm** [comment]
bot, build

**Pull request merged**